### PR TITLE
Addition of choice between LocalResponseNormalization and BatchNormalization and command line based control of train.py file.

### PR DESCRIPTION
**Pull request by jatinmandav**

### CODE REVIEW

**Comment by Borda** [comment]
Hi, it seems that this repo is inactive for a while... (more than a year :worried:)
Would you consider to pass your changes to this fork https://github.com/Borda/keras-yolo3 ?
